setup.cfg

```diff
@@ -18,11 +18,11 @@ precision = 2
 
 [isort]
 not_skip = __init__.py
-known_third_party = appdirs, backports, faker,factory, faker, freezegun, future, hamster_lib, icalendar, past, pytest, pytest_factoryboy,
+known_third_party = appdirs, configparser, faker,factory, faker, freezegun, future, hamster_lib, icalendar, past, pytest, pytest_factoryboy,
```

Comment:
I agree. To keep this hotfix branch clean and focused this has been place as a seperate [issue](https://projecthamster.atlassian.net/browse/LIB-256).